src/main/java/com/google/cloud/spanner/pgadapter/utils/Metrics.java

```diff
@@ -31,31 +30,42 @@ public class Metrics {
   static final String PGADAPTER_LATENCY_DESCRIPTION =
       "Latency between PGAdapter receiving a statement from the client and PGAdapter returning the last row of the response to the client";
 
-  private final LongHistogram spannerClientLibLatencies;
-  private final LongHistogram pgadapterLatencies;
+  private final DoubleHistogram spannerClientLibLatencies;
+  private final DoubleHistogram pgadapterLatencies;
+
+  public static List<Double> getMetricLatencyMillisBuckets() {
```

Comment:
nit: does this need to be public?

Reply:
Yes, we need this in the tpcc benchmark code - pgadapter/benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/Metrics.java.